gradio/blocks.py

```diff
@@ -233,13 +236,19 @@ def get_config(self):
         for parameter in signature.parameters.values():
             if hasattr(self, parameter.name):
                 value = getattr(self, parameter.name)
-                config[parameter.name] = utils.convert_to_dict_if_dataclass(value)
+                if isinstance(value, Callable):
+                    continue
+                if dataclasses.is_dataclass(value):
```

Comment:
I think the better fix is to use `default=str` in the `orjson.dumps` call in `process_msg`. There could be non-serializable values beyond functions/dataclasses set in state variables/custom components.

Reply:
Thanks, fixed